README.md

```diff
@@ -22,8 +22,8 @@ Then, you need to add some informations in your configuration files
 ```php
 'providers' => [
     ...
-    Mailjet\LaravelMailjet\MailjetServiceProvider::class,
-    Mailjet\LaravelMailjet\MailjetMailServiceProvider::class,
+    Mailjet\LaravelMailjet\Providers\MailjetServiceProvider::class,
```

Comment:
Good, I'll leave the old facade and provider as it was and let the user choose between facade/contract approach.